org.wxhexeditor.wxHexEditor.appdata.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.wxhexeditor.wxHexEditor</id>
+  <metadata_license>CC-BY-SA-3.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <name>wxHexEditor</name>
+  <summary>Advanced large file hex editor</summary>
+  <summary xml:lang="de_DE">Umfangreicher Hex-Editor für große Dateien</summary>
+  <description>
+    <p>Edit large files and directly edit block devices.</p>
+    <p xml:lang="de_DE">Bearbeiten sie große Dateien oder Massenspeichergeräte direkt auf Byte-Level.</p>
+    <p>Advanced Features include:</p>
+    <ul>
+      <li>editing very large files (bigger than RAM)</li>
+      <li>directly editing block devices (data recovery)</li>
+      <li>compare files</li>
+      <li>directly edit XOR encrypted files</li>
+      <li>colorized tags</li>
+    </ul>
+    <p xml:lang="de_DE">Besondere Funktionen:</p>
+    <ul xml:lang="de_DE">
+      <li>Bearbeiten von sehr großen Dateien (größer als RAM)</li>
+      <li>direktes Bearbeiten von Massenspeichergeräten (für z.B. Datenrettung)</li>
+      <li>Vergleichen von Dateien</li>
+      <li>direktes bearbeiten XOR chiffrierter Dateien</li>
+      <li>eingefärbte Markierungen</li>
+    </ul>
+  </description>
+  
+  <launchable type="desktop-id">org.wxhexeditor.wxHexEditor.desktop</launchable>
+  
+  <screenshots>
+    <screenshot type="default" width="913" height="662">
+      <image>
+           http://www.wxhexeditor.org/images/screenshot.png
+        </image>
+      <caption/>
+    </screenshot>
+    <screenshot width="641" height="511">
+      <image>http://wiki.wxhexeditor.org/images/8/8a/XOR_ViewThru.gif</image>
+      <caption>XOR overlay mode</caption>
+    </screenshot>
+    <screenshot width="1094" height="449">
+      <image>http://wiki.wxhexeditor.org/images/5/56/Compare_Files.png</image>
+      <caption>diff compare mode</caption>
```

Comment:
`appstream-glib` complains about this caption:
```
• style-invalid         : <caption> requires sentence case [diff compare mode]
```

It should be:
```
      <caption>Diff compare mode</caption>
```

Next time try to to validate this file before committing it to the repo:
```
flatpak run org.freedesktop.appstream-glib validate-strict "org.wxhexeditor.wxHexEditor.appdata.xml"
```